test/e2e/lifecycle.go

```diff
@@ -84,6 +85,39 @@ func TearDown(client *Client) {
 	}
 }
 
+func (c *Client) LogsFor(namespace, name string, gvr schema.GroupVersionResource) (string, error) {
+	// Get all pods in this namespace.
+	pods, err := c.Kube.Kube.CoreV1().Pods(namespace).List(metav1.ListOptions{})
+	if err != nil {
+		return "", err
+	}
+
+	logs := "" //map[string]string{}
```

Comment:
Move down to line 110.

Or use the map and get the logs for every matching Pod.

Reply:
this e2e code is wip.

Reply:
OK...is the overall PR WIP or will it be merged with WIP e2e code as-is?